reference/commands/new.rst

```diff
@@ -50,7 +50,7 @@ The available templates are:
   Creates a simple recipe with some example code and helpful comments,
   and is a good starting point to avoid writing boilerplate code.
 
-  Its variables are: [name], [version], [description], [require, require...]
+  Its variables are: [name], [version], [description], [requires, requires...], [tool_requires, tool_requires, ...]
```

Comment:
why is it doubled here?

Reply:
So users know that it is a list that they can pass multiple values multiple times, but if you had to ask, then maybe it's not as obvious as I had hoped for? :/

Reply:
maybe it needs to be `[requires1, requires2...requiresN]` then?
otherwise, it looks like I need to pass the same thing twice